<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CSimpleClass</class>
 <widget class="QMainWindow" name="CSimpleClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CSimple</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>240</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>菜单</string>
    </property>
    <addaction name="actionClose"/>
    <addaction name="actionHide"/>
    <addaction name="actionShow"/>
    <addaction name="actionMm1"/>
    <addaction name="actionMm2"/>
    <addaction name="actionVisible"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionClose">
   <property name="text">
    <string>关闭</string>
   </property>
  </action>
  <action name="actionHide">
   <property name="text">
    <string>隐藏</string>
   </property>
  </action>
  <action name="actionShow">
   <property name="text">
    <string>显示</string>
   </property>
  </action>
  <action name="actionMm1">
   <property name="text">
    <string>mm1</string>
   </property>
  </action>
  <action name="actionMm2">
   <property name="text">
    <string>mm2</string>
   </property>
  </action>
  <action name="actionVisible">
   <property name="text">
    <string>Visible</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="csimple.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionClose</sender>
   <signal>triggered()</signal>
   <receiver>CSimpleClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHide</sender>
   <signal>triggered()</signal>
   <receiver>CSimpleClass</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShow</sender>
   <signal>triggered()</signal>
   <receiver>CSimpleClass</receiver>
   <slot>showFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
